spec.bs

```diff
@@ -2,18 +2,17 @@
 Title: Private State Token API
 H1: Private State Token API
 Shortname: private-state-token-api
-Level: 1
 Status: CG-DRAFT
 Group: WICG
 Repository: WICG/trust-token-api
 URL: https://wicg.github.io/trust-token-api/
-Editor: Aykut Bulut, Google https://www.google.com/, aykutb@google.com
-Editor: Steven Valdez, Google https://www.google.com/, svaldez@google.com
-Abstract: Private State Token API is a web platform API that allows propagating a limited amount of anti-fraud signals across sites, using the Privacy Pass protocol as an underlying primitive.
+Editor: Aykut Bulut, Google LLC https://www.google.com/, aykutb@google.com
+Editor: Steven Valdez, Google LLC https://www.google.com/, svaldez@google.com
```

Comment:
I asked about this internally a while ago and I think the result was that just "Google" is fine.

Reply:
Ah, sure - that works too (I was given different advice by someone on the web standards team - shrug).